Unsorted/24292844_172adec.txt

```diff
@@ -11,6 +11,11 @@
 		釣る側と釣られる側の<end_line>
 		知恵比べ<end_line>
 	</sjis>
+	<ascii>
+		Fishing is life itself<end_line>
+		A battle of wisdom between<end_line>
+		the fisher and the fish<end_line>
```

Comment:
To my understanding the fisherman is making a comparison between fishing and life, as in an existential question. The first line here doesn't convey that too well. Here it implies fishing is similar to life, so we would think to apply traits of "life" to fishing, when it is the opposite point that the fisherman is attempting to make.

Reply:
I see, thank you for the explanation. Even though I wrote in english, I was thinking in portuguese, where when phrased it like that it implies they have traits in common,  regardless of which side you're applying the traits from. It seems I have some stuff to learn. Thank you, I'll rewrite it.

Reply:
I rephrased as :
> Life is a lot like fishing.<end_line> 
> A battle of wisdom between<end_line>
> the fisher and the fish.<end_line>

commit: 64895e7